### ISSUE TITLE
TrackerControl is invisible on Windows platform

### ISSUE THREAD

**Issue by lu8848** [oxyplot/oxyplot-maui#13]
Tested with CustomTrackerPage sample, the oxy:TrackerControl is invisible for .net 8.0 and run on Windows platform.

It is visible for .net 8.0 on Android.

It is also visible for .net 7.0 on Windows.

**Comment by janusw** [comment]
> Tested with CustomTrackerPage sample, the oxy:TrackerControl is invisible for .net 8.0 and run on Windows platform.
> 
> It is visible for .net 8.0 on Android.

I can confirm that it's visible on Android, and I also see it on iOS (a grey box that show various values, appearing when you tap a point on the graph).

Have not tested Windows, but will try to do that soon ...

**Comment by janusw** [comment]
I managed to test the sample app on Windows by now, with the following results ...

Running the sample app with net7.0 target (at the tagged version `1.0.0`):
* Single-finger panning works fine on all examples I tried. ✔️ 
* Two-finger scrolling does not work. ⚡ 
* On the `CustomTrackerPage`, I see the `TrackerControl` working well (by clicking on one of the graphs). ✔️ 
* On the example "Annotations -> Tool tips" in the example browser, the tool tips work well (clicking on one of the objects). ✔️ 

Running the sample app with net8.0 target (at commit 707a12c1e724b66276cdc827c032297bfdc76d45):
* Single-finger panning works fine on all examples I tried. ✔️ 
* Two-finger scrolling works as well. ✔️ 
* On the `CustomTrackerPage`, I see that the `TrackerControl` is not showing up properly (by clicking on one of the graphs). ⚡ 
* On the example "Annotations -> Tool tips" in the example browser, the tool tips do not show up (clicking on one of the objects). ⚡ 

So, mixed results: scrolling only works with net8.0, while `TrackerControl` and tool tips only work with 7.0. Kinda strange, but confirming your observsations, @lu8848.

Actually, when looking more closely at the `CustomTrackerPage` with net8.0, it seems like sometimes whitish text is overlaying on the graph (which is mostly invisible on the white background, and only shows around the line of the graph). So probably the `TrackerControl` is shown after all, but just colored badly ...?!?
